doc/source/whatsnew/v0.18.1.txt

```diff
@@ -29,6 +29,56 @@ New features
 Enhancements
 ~~~~~~~~~~~~
 
+Method chaininng improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Following methods / ``Indexer`` now accept ``callable`` as input. It is
+intended to make these more useful in the method chain.
+(:issue:`11485`, :issue:`12533`)
+
+- ``.where`` and ``.mask``
+- ``.query``
+- ``.loc``, ``iloc`` and ``.ix``
```

Comment:
do:

``.where()`` and ``.mask()``
``.query()``
``.loc[]``, ``.iloc[]``, ``.ix[]``